docs/api/synthetics/params/add-param.asciidoc

```diff
@@ -0,0 +1,123 @@
+[[add-parameters-api]]
+== Add Parameters API
+++++
+<titleabbrev>Add Parameters</titleabbrev>
+++++
+
+Adds one or more parameters to the synthetics app.
+
+=== {api-request-title}
+
+`POST <kibana host>:<port>/api/synthetics/params`
+
+`POST <kibana host>:<port>/s/<space_id>/api/synthetics/params`
+
+=== {api-prereq-title}
+
+You must have `all` privileges for the *Synthetics* feature in the *{observability}* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+
+[[parameters-add-request-body]]
+==== Request body
+
+The request body can contain either a single parameter object or an array of parameter objects. The parameter object schema includes the following attributes:
+
+`key`::
+(Required, string) The key of the parameter.
+
+`value`::
+(Required, string) The value associated with the parameter.
+
+`description`::
+(Optional, string) A description of the parameter.
+
+`tags`::
+(Optional, array of strings) An array of tags to categorize the parameter.
+
+`share_across_spaces`::
+(Optional, boolean) Whether the parameter should be shared across spaces.
+
+When adding a single parameter, provide a single object. When adding multiple parameters, provide an array of parameter objects.
+
+[[parameters-add-example]]
+==== Example
+
+Here are examples of POST requests to add parameters, either as a single parameter or as an array of parameters:
+
+To add a single parameter:
+
+[source,json]
+--------------------------------------------------
+{
+  "key": "api_key",
```

Comment:
```suggestion
  "key": "your-param-key",
```

I detailed my rationale for suggesting this in a comment further down in the file.